pr_agent/tools/pr_code_suggestions.py

```diff
@@ -39,7 +39,7 @@ def __init__(self, pr_url: str, cli_mode=False):
                                           settings.pr_code_suggestions_prompt.user)
 
     async def suggest(self):
-        assert type(self.git_provider) == GithubProvider, "Only Github is supported for now"
+        assert type(self.git_provider) != BitbucketProvider, "Bitbucket is not supported for now"
```

Comment:
**Suggestion:** In the `suggest` function, the assertion message is not very informative. It would be better to provide a more descriptive message to help the user understand why Bitbucket is not supported.
```suggestion
    async def suggest(self):
        assert type(self.git_provider) != BitbucketProvider, "Bitbucket is currently not supported due to lack of API support for code suggestions."
```